sdk/lib/rtl/dbgbuffer.c

```diff
@@ -14,30 +15,80 @@
 
 /* FUNCTIONS *****************************************************************/
 
+PVOID
+NTAPI
+RtlpDebugBufferCommit(_Inout_ PRTL_DEBUG_INFORMATION Buffer,
+                     _In_ SIZE_T Size)
+{
+    ULONG Remaining = Buffer->CommitSize - Buffer->OffsetFree;
+    PVOID Result;
+    NTSTATUS Status;
+
+    if (Size > 0x7ffffff)
```

Comment:
```suggestion
    if (Size > MAXLONG)
```

Reply:
Yeah I just didnt want it overflowing so I chose a big number, guess this will work as well.